prac_05/state_names.py

```diff
@@ -15,3 +15,6 @@
     else:
         print("Invalid short state")
     state_code = input("Enter short state: ")
+
+for short_state, long_state in CODE_TO_NAME.items():
+    print(f"{short_state:3} is {long_state}")
```

Comment:
Could these two lines of code be reformatted into a separate function? eg. print_state_codes() (not asked in question but thought I would include anyway). The loop above this could also be refactored, eg. get_state_name().

Reply:
I like your thinking :)